### DISCUSSION TITLE
Cheat sheets

### DISCUSSION THREAD

**Discussion by peter-csala** [App-vNext/Polly#1823] [Ideas]
What if we create cheat sheets for each strategy?

I've put two examples together to depict my idea. 
_I'm not really good at UI design so apologize if it looks a bit dull / unprofessional_.

[Fallback cheat sheet.pdf](https://github.com/App-vNext/Polly/files/13443877/Fallback.cheat.sheet.pdf)
<img width="975" alt="Screenshot 2023-11-22 at 20 52 27" src="https://github.com/App-vNext/Polly/assets/57183693/35ce7760-8687-4bfc-b218-1c13a7a416d4">

[Timeout cheat sheet.pdf](https://github.com/App-vNext/Polly/files/13443878/Timeout.cheat.sheet.pdf)
<img width="974" alt="Screenshot 2023-11-22 at 20 52 49" src="https://github.com/App-vNext/Polly/assets/57183693/1163ca88-a140-4d11-ace5-8af8cb5095a9">

Or do we have any template what we could use?

What do you guys think?
cc: @martintmk, @martincostello, @joelhulen

**Comment by martincostello** [comment]
I like the idea of these little copy-paste samples to just take and go code with, without the need for the extra explanatory comments.

These remind me of the posters that used to accompany new .NET Framework releases showing all the new APIs.
